forms-flow-api/src/formsflow_api/services/application.py

```diff
@@ -59,6 +59,7 @@ def create_application(data, token, **kwargs):
                     "formName": {"value": mapper.form_name},
                     "submitterName": {"value": application.created_by},
                     "submissionDate": {"value": application.created.__str__()},
+                    "tenantKey": {"value": mapper.tenant},
```

Comment:
are we doing the null check here / This change is only needed for multitenant env right?